sites/target.py

```diff
@@ -0,0 +1,199 @@
+        in_stock = False
+
+        self.browser.get(self.product)
+
+        while not in_stock:
+            try:
+                if not img_found:
+                    product_img = self.browser.find_elements_by_class_name('slideDeckPicture')[0].find_element_by_tag_name(
+                        "img")
+                    self.image_signal.emit(product_img.get_attribute("src"))
+                    self.product_image = product_img.get_attribute("src")
+                    img_found = True
+            except Exception as e:
+                continue
+
+            try:
+                ship_btn = self.browser.find_element_by_xpath('//button[@data-test= "shipItButton"]')
+                self.browser.execute_script("return arguments[0].scrollIntoView(true);", ship_btn)
+                ship_btn.click()
+                in_stock = True
+                self.status_signal.emit(self.create_msg("Added to cart", "normal"))
+            except Exception as e:
+                self.status_signal.emit(self.create_msg("Waiting on Restock", "normal"))
+                self.browser.refresh()
+
+    def atc(self):
+        declined_ins = False
+        at_checkout = False
+
+        self.status_signal.emit(self.create_msg("Declining Insurance", "normal"))
+
+        while not declined_ins:
+            try:
+                decline_ins_btn = self.browser.find_element_by_xpath('//button[@data-test= "espModalContent-declineCoverageButton"]')
+                decline_ins_btn.click()
+                declined_ins = True
+            except:
+                continue
+
+        self.status_signal.emit(self.create_msg("Viewing Cart before Checkout", "normal"))
+
+        while not at_checkout:
+            try:
+                checkout_btn = self.browser.find_element_by_xpath('//button[@data-test= "addToCartModalViewCartCheckout"]')
+                checkout_btn.click()
+                at_checkout = True
+            except:
+                continue
+
+    def checkout(self):
+        did_checkout = False
+        self.status_signal.emit(self.create_msg("Checking out", "normal"))
+
+        while not did_checkout:
+            try:
+                self.browser.find_element_by_xpath('//button[@data-test= "checkout-button"]').click()
+                did_checkout = True
+                time.sleep(random_delay(self.monitor_delay, settings.random_delay_start, settings.random_delay_stop))
+            except:
+                continue
+
+
+    def submit_billing(self):
+        added_cc = False
+        added_cvv = False
+
+        self.status_signal.emit(self.create_msg("Entering CC #", "normal"))
+
+        while not added_cc:
+            try:
+                cc_input = self.browser.find_element_by_id("creditCardInput-cardNumber")
+                cc_input.send_keys(self.profile["card_number"])
+                self.browser.find_element_by_xpath('//button[@data-test= "verify-card-button"]').click()
+                added_cc = True
+            except:
```

Comment:
When I tested this the program executed too fast to catch the CVV popup and thus advanced to submit_order too early. I recommend adding a small delay here. Perhaps 2-3 seconds.
```suggestion
                added_cc = True
                time.sleep(2)
            except:
```